src/VisualStudio/CSharp/Test/GlyphExtensionsTests.cs

```diff
@@ -169,6 +170,14 @@ public void TestWithEventsMemberGlyph()
                 isWithEvents: true);
         }
 
+        [Fact, WorkItem(26642, "https://github.com/dotnet/roslyn/issues/26642")]
+        public void TestNoReferenceToImageCatalog()
```

Comment:
We probably should have this in EditorServicesTests somewhere. If you need to make a new class, that's fine.

Reply:
We also probably should have a comment saying why this is important...